helm/rook-operator/charts/monitoring/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.global.rookEnabled }}
+{{- if .Values.global.isManagementCluster }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    cluster_id: {{ .Values.codename }}
+    team: rook
+    operator: rook
+    storage-backend: ceph
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+  name: rook-ceph-mgr
+  namespace: {{ .Values.codename }}-prometheus
```

Comment:
this is a bit shitty, but once Atlas open up the namespaces you can create servicemonitors in then this can be changed to the same namespace as the Rook installation uses.